modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+                    "loggingRules": {
+                        "agents.partners": {
+                        "bucket": "conflictlogsbucket2",
+                        "collection": "myappscope.partnerlogs"
+                    },
+                    "agents.retailers": null,
+                    "inventory": {
+                        "bucket": "conflictlogsbucket2",
+                        "collection": "myappscope.inventorylogs"
+                    }
+            }
+}'
+----
+
+[#disabled-conflict-logging]
+==== Disabled
+
+To disable conflict logging, set `--data-urlencode conflictLogging='{}'`.
+
+[#logging-rules]
+=== loggingRules
+
+Use `loggingRules` to override the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#enabled-for-all-conflicts[default conflict logging settings].
+
+[NOTE]
+====
+* The keys and their priorities of each KV pair are specified in the format similar to xref:manage:manage-xdcr/replicate-using-scopes-and-collections.adoc##rules-for-explicit-mappings[Rules for Explicit Mappings] and also have the same priority as listed in the mapping rules.
+
+* Similar to explicit mapping mode, if a source scope or collection is not specified, then the conflict from that source is ignored.
+
+* Conflict logging rules can be saved without enabling the Conflict Logging setting, however, the rules will come into effect only when the Conflict Logging setting is enabled.
+====
+
+IMPORTANT: If logging fails, XDCR continues with conflict resolution to ensure High Availability.
+
+The following section explains the mapping of conflict logs to their source.
+The value of `loggingRules` is a JSON object composed of any number of the following Key-Value (KV) formats.
+
+|===
+| Key | Value | Description 
+
+| [scope]
+| `{}`
+
+Empty JSON object.
+| All mutations from the specified scope will be logged in the conflict collection defined by bucket, scope, and collection in the *ConflictLogging* key.
+
+| [scope]
+| `null`
+| All mutations from the specified scope will not be logged.
+
+| [scope.collection]
+| `{}`
+| All mutations from the specified collection will be logged in the conflict collection defined by bucket, scope, and collection in the *ConflictLogging* key.
+
+| [scope.collection]
+| `null`
+| All mutations from the specified collection will not be logged.
+
+| [scope]
+|`{
+   “bucket”: [otherBucket],
+   “collection”: [otherScope].[otherCollection]”
+  }`
+| All mutations from the chosen scope will be logged in the specific conflict collection defined by otherBucket, otherScope, and otherCollection; instead of the original location mentioned in the *ConflictLogging* key.
+
+| [scope.collection]
+|`{
+   “bucket”: [otherBucket],
+   “collection”: [otherScope].[otherCollection]”
+  }`
+| All mutations from the chosen scope will be logged in the specific conflict collection defined by bucket, scope, and collection; instead of the original location mentioned in the *ConflictLogging* key.
```

Comment:
All conflicts, not mutations.
conflictLogging key, not ConflictLogging key.
There's "bucket" and "collection" -- but the sentence mentions "scope" as well.
And, looks like this rule is for a collection, not a scope.
To be consistent with the other descriptions, should be:

All conflicts from the chosen collection will be logged in the specific conflict collection defined by otherBucket, otherScope, and otherCollection; instead of the original location mentioned in the *conflictLogging* key.